ci/infra/testrunner/tests/test_cilium.py

```diff
@@ -52,3 +52,27 @@ def test_cillium(deployment, kubectl):
         time.sleep(30)
 
     assert all_reachable
+
+    logger.info("Change Cilium ConfigMap and set MTU 1450 bytes")
+    # Change Cilium ConfigMap and set MTU 1450 bytes
+    kubectl.run_kubectl("get configmap -n kube-system cilium-config -o json | jq '.data.mtu = "1450"'")
```

Comment:
```suggestion
    kubectl.run_kubectl("get configmap -n kube-system cilium-config -o json | jq '.data.mtu = \"1450\"'")
```

to fix this error:

```
/usr/lib64/python3.6/ast.py:35: in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
E     File "/home/jenkins/workspace/caasp-jobs_pr_test_PR-1404/ci/infra/testrunner/tests/test_cilium.py", line 58
E       kubectl.run_kubectl("get configmap -n kube-system cilium-config -o json | jq '.data.mtu = "1450"'")
E                                                                                                     ^
E   SyntaxError: invalid syntax
```

Reply:
Thank you Michu, let's see if this will help. Yeah!